README.md

```diff
@@ -54,11 +54,11 @@ Of course you can also add a totally new exercise, but it might be a good idea t
 
 #### Updating an exercise test suite
 Updating a test suite of an existing exercise is special because it usually affects all languages.
-You can refer to the [documentation on updating a test suite](https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#updating-an-exercise-test-suite).
+You can refer to the [documentation on updating a test suite](https://github.com/exercism/problem-specifications/blob/master/CONTRIBUTING.md#updating-an-exercise-test-suite).
 Note that the whole test suite must run with the sample solution within a couple of seconds.
 
 ### Repository structure and conventions
-The [track anatomy documentation](https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#track-anatomy) is a general description of all the files and directories that are not explicitly described below.
+The [track anatomy documentation](https://github.com/exercism/problem-specifications/blob/master/CONTRIBUTING.md#track-anatomy) is a general description of all the files and directories that are not explicitly described below.
```

Comment:
If advisement of a new file in the docs repo is received in https://github.com/exercism/haskell/pull/578/files#r127601273 , then this link can be updated. Until then, it has to stay in problem-specifications CONTRIBUTING guide.